app/src/main/res/xml/preferences_master.xml

```diff
@@ -65,6 +66,14 @@
                 android:key="@string/pref_key__app_start_folder"
                 android:summary="%1$s"
                 android:title="@string/app_start_folder" />
+            <android.support.v7.preference.EditTextPreference
+                android:defaultValue="@string/empty_string"
+                android:icon="@drawable/ic_iconfinder_ic_vpn_key_48px_3669407"
+                android:inputType="textPassword"
```

Comment:
Now the password will be stored by the keyboard for autocompletion :-(

Reply:
inputType seems to be utterly broken, well done supportlibs..
Can you change to the working flags/types again?

Reply:
Suggestions are default by almost every keyboards. I fixed it and removed the unnecessary ASTERISK-String.

Reply:
Yeah but for every I used till date, it only chcecks for contact names out of addressbook; for "official words book" and for manually added/confirmed word.  
Automatically adding it sounds malware and keylogger functionalities to me :D